### DISCUSSION TITLE
Cloud banner alignment with rest of product

### DISCUSSION THREAD

**Discussion by staton-hyse11** [grafana/grafana#62784] [Saga Design System]
Currently, Grafana Cloud has a very differently styled banner that breaks the pattern of our buttons by theme and general styling pattern for the alerts.

![image](https://user-images.githubusercontent.com/101748563/216346729-47b6d446-05f2-4e5f-a5de-8f116b602fac.png)

Without proposing a solution, I'd like to recommend that we begin the effort to have this type of alert better align with the others (toast and inline banner) for both styling consistency and theme consistency (we don't allow for outline buttons to have border change and the warning banner has a light theme button and iconbutton — i.e. dark text vs white text). 

@JoaoSilvaGrafana @natydej

**Comment by amy-super** [comment]
I believe this was worked on a while ago. Before proposing any changes, I'd be curious to see if @aligerrard and her team have done any metric-gathering on these banners and whether they are helpful/effective for our users. If they are working well, it might be worth discussing what specifically is working well before aligning them with the rest of our alert patterns.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This looks to be the start of an ongoing discussion, and we'd like discussions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) when you have the solution.
